test/test_functional.py

```diff
@@ -389,6 +394,16 @@ def test_linearity_of_istft4(self):
         data_size = (2, 7, 3, 2)
         self._test_linearity_of_istft(data_size, kwargs4, atol=1e-5, rtol=1e-8)
 
+    def test_batch_istft(self):
+
+        stft = torch.tensor([
+            [[4., 0.], [4., 0.], [4., 0.], [4., 0.], [4., 0.]],
+            [[0., 0.], [0., 0.], [0., 0.], [0., 0.], [0., 0.]],
```

Comment:
Why repeat all 0s instead of something more interesting (maybe just ```[0., 4.]```)

Reply:
I'm reusing the parameters from `test_istft_of_ones`.